cpp/benchmarks/join/nvbench_helpers.hpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <benchmarks/join/join_common.hpp>
+
+#include <cudf/types.hpp>
+
+#include <nvbench/nvbench.cuh>
+
+NVBENCH_DECLARE_ENUM_TYPE_STRINGS(
+  cudf::null_equality,
+  [](auto value) {
+    switch (value) {
+      case cudf::null_equality::EQUAL: return "EQUAL";
+      case cudf::null_equality::UNEQUAL: return "UNEQUAL";
```

Comment:
Nit: Would the following be clearer?
```suggestion
      case cudf::null_equality::EQUAL: return "NULLS_EQUAL";
      case cudf::null_equality::UNEQUAL: return "NULLS_UNEQUAL";
```

Reply:
I chose `EQUAL` and `UNEQUAL` here to mimic the `cudf::null_equality` enum. AFAICT, nvbench enum type strings defined in `benchmarks/io/nvbench_helpers.hpp` match the enums that we have in libcudf, so I followed a similar pattern in the join nvbench helpers as well.